code/game/objects/items/weapons/dice.dm

```diff
@@ -49,6 +51,16 @@
 	icon_state = "d4"
 	sides = 4
 
+/obj/item/weapon/dice/d6
+	name = "d6"
+
+/obj/item/weapon/dice/fudge
+	name = "fudge die"
+	desc = "A die with six sides but only three results. Is this a plus or a minus? Your mind is drawing a blank..."
+	sides = 3 //shhh

```

Comment:
it's a six sided die with each side represented twice. This ain't no barrel dice crap.